internal/core/api.rs

```diff
@@ -158,6 +177,33 @@ impl PhysicalSize {
     }
 }
 
+/// The requested size to render the window.
+#[derive(Clone, Debug, derive_more::From, PartialEq)]
+pub enum RequestedSize {
```

Comment:
In the meeting, we said `WindowSize` and `WindowPosition`  
Requested doesn't look too bad either in this case.

Reply:
Yes, I liked Requested* better and thought I'd try that.

It is easy enough to change now! I Love language servers and the renaming they support:-)